<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setwindow</class>
 <widget class="QWidget" name="setwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>193</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>75</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_14">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>设备类型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_6">
        <property name="maximumSize">
         <size>
          <width>71</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>0xa1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0xa2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0xa3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0xa4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0xa5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0xa6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0xa7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0x18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0x19</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>协议类型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_8">
        <property name="maximumSize">
         <size>
          <width>71</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>SFP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RSSP-I</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>透传数据</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>通信周期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_7">
        <property name="maximumSize">
         <size>
          <width>71</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>50ms</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100ms</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>200ms</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>134</width>
     <height>182</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>串口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="com1_serialBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>波特率</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="com1_baudBox">
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>56000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>128000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>256000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>512000</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>数据位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="com1_dataBox">
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>校验位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="com1_parityBox">
        <item>
         <property name="text">
          <string>0x00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0x01</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0x02</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>停止位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="com1_stopBox">
        <item>
         <property name="text">
          <string>0x00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0x01</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0x02</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:qradialgradient(spread:pad,cx:0.5,cy:0.5,radius:0.5,fx:0.5,fy:0.5,stop:0 rgba(255,0,0,255),stop:1 rgba(255,255,255,255));
border-radius:12px;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>打开串口</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
